cmd/key-helper/importKey.go

```diff
@@ -140,7 +145,89 @@ func wrapPrivateKey(wrappingKey string, privateKeyPath string) (string, error) {
 	return base64Ciphertext, nil
 }
 
-func importKeyToTransit(ctx context.Context, client *vault.Client, ciphertext string, keyName string) error {
+// Checks PrivateKey type and bitlength in EC, PKCS1 and PKCS8 containers
+func checkKeyType(keyPath string) (string, string, error) {
+	// Key types supported by Notary project specification
+	supportedRSAKeys := map[int]string{
+		2048: "rsa-2048",
+		3072: "rsa-3072",
+		4096: "rsa-4096",
+	}
+	supportedECDSAKeys := map[string]string{
+		"P-256": "ecdsa-p256",
+		"P-384": "ecdsa-p384",
+		"P-521": "ecdsa-p521",
+	}
+	digestAlgorithm := map[string]string{
```

Comment:
Just to be clear, this is the mapping I think should be removed; the above two should stay.